uisdk/src/test/java/com/karhoo/uisdk/screen/booking/checkout/view/CheckoutViewPresenterTests.kt

```diff
@@ -245,10 +245,9 @@ class CheckoutViewPresenterTests {
     fun `passenger details visible and valid but payment is not`() {
```

Comment:
do these tests make sense anymore given the fact a valid payment pre-auth is no longer mandatory to proceed?